components/ColorDetails.js

```diff
@@ -40,8 +43,8 @@ export const ColorDetail = ({ color }) => {
     { key: "HWB", value: fullColor.tohwb() },
     { key: "CMYK", value: fullColor.tocmyk() },
     { key: "CIELAB", value: fullColor.tolab() },
-    { key: "Luminance", value: (fullColor.luminance() * 100).toFixed(2) + "%" },
-    { key: "Darkness", value: (fullColor.darkness() * 100).toFixed(2) + "%" }
+    { key: t("Luminance"), value: (fullColor.luminance() * 100).toFixed(2) + "%" },
```

Comment:
I don't think this needs translation.

Reply:
Done.

Reply:
Here are the screenshots. App looks the same since there hasn't been any other languages.
![Screenshot from 2022-11-09 16-01-11](https://user-images.githubusercontent.com/13333613/200865968-bbc9b38e-9e51-44e4-84eb-9b7dd8c8deb1.png)
![Screenshot from 2022-11-09 16-01-05](https://user-images.githubusercontent.com/13333613/200865980-da63b3db-3b61-4d06-bc72-187cb63d08eb.png)
![Screenshot from 2022-11-09 16-00-59](https://user-images.githubusercontent.com/13333613/200865984-767fcb9e-9ca7-4f4b-87a7-77120b5e5b80.png)
![Screenshot from 2022-11-09 16-00-52](https://user-images.githubusercontent.com/13333613/200865986-078ec475-8ddb-418c-a4f9-32cd066866a3.png)